### ISSUE TITLE
Description and Keywords not added to header

### ISSUE THREAD

**Issue by jason-unleashed** [steamlead/grav-plugin-autoseo#2]
I don't have the best grip on SEO and tags, so if I'm reading this wrong please let me know.

It looks like the default description and headers are added as "property" instead of "name".  When I do an SEO analysis at Bing it tells me that I'm missing the description.

When I edit a specific page and manually add a description in the metadata field the "property=description ***" disappears.

When I disable autoseo op a specific page, I see the following meta tags:
  `<meta name="description" content="Unleashed Financial LLC, providing affordable financial planning and investment management for young professionals and growing families " />`
**[this is the data I entered for my site description in the site.yaml]**

When I enable autoseo I see the following meta tags:
  ```
<meta property="description" content="The same principles to investing money apply.If I asked you how you spend your money, you could answer with a decent amount of precision (and maybe some embarrassment). But if" />
<meta property="keywords" content="articleseducationplanning" />
<meta name="og:sitename" property="og:sitename" content="Unleashed Financial LLC" />
<meta name="og:title" property="og:title" content="investing-your-time-vs-spending-your-time" />
<meta name="og:type" property="og:type" content="article" />
<meta name="og:url" property="og:url" content="https://www.unleashedfinancial.com/articles/investing-your-time" />
<meta name="og:description" property="og:description" content="The same principles to investing money apply.If I asked you how you spend your money, you could answer with a decent amount of precision (and maybe some embarrassment). But if" />
<meta name="og:image" property="og:image" content="https://www.unleashedfinancial.com/user/pages/06.articles/Investing-your-Time/Investing Your Time.jpg" />
<meta name="twitter:card" property="twitter:card" content="summary_large_image" />
<meta name="twitter:title" property="twitter:title" content="investing-your-time-vs-spending-your-time" />
<meta name="twitter:description" property="twitter:description" content="The same principles to investing money apply.If I asked you how you spend your money, you could answer with a decent amount of precision (an" />
<meta name="twitter:image" property="twitter:image" content="https://www.unleashedfinancial.com/user/pages/06.articles/Investing-your-Time/Investing Your Time.jpg" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
```
Is this a bug or am I implementing something wrong?
Thank,
J

**Comment by LaurentOngaro** [comment]
Hi Jason, 
I thing you are right !
The meta tag field were wrongly named "property" instead of "name". 
I'll upload a new version to fix that bug in a few minutes.
thanks for your report.

**Comment by LaurentOngaro** [comment]
the source code has been updated.
the issue has been fixed in 1.3.2 release that I've just posted. 
You'll have to wait a few for the new release to be published.
